sw-precache-config.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
+  staticFileGlobs: [
+    'out/**.html',
```

Comment:
I noticed that the demo from the sw-precache repository also uses the same pattern, as in [this example](https://github.com/GoogleChrome/sw-precache/blob/master/demo/sw-precache-config.js#L14). So, this should be fine for now.

---

From the [glob implementation](https://www.npmjs.com/package/glob) (emphasis mine):

> `**` If a "globstar" is **alone in a path portion**, then it matches zero or more directories and subdirectories searching for matches.

Since the globstar is not alone in the path portion (which would look like `out/**`), so it falls back to the same behavior as the single-star which is to only match the files.